fury/tests/test_actors.py

```diff
@@ -1819,3 +1820,84 @@ def test_actors_primitives_count():
         primitives_count = test_case[2]
         act = act_func(**args)
         npt.assert_equal(primitives_count_from_actor(act), primitives_count)
+
+def test_kde_actor(interactive = True):
```

Comment:
Not sure if this should be tested here, as you don't have a `kde` actor definition on `actor.py`, maybe take a look at `actors/peak.py` and `peak` in `actor.py`, to see how you could adjust that.

Reply:
Ok, I will be looking into that :+1